### ISSUE TITLE
Possible on dired_jump to not automatically set the project folder?

### ISSUE THREAD

**Issue by excetara2** [aziz/SublimeFileBrowser#117]
I am just curious if this is possible to use dired_jump and not automatically set the project folder. If you dired_jump to `/Volumes` for example on mac. Then it bogs down the system a ton I realized because it's trying to load all these files. Or don't automatically add the folder if it is larger than a certain size or if it's outside of the home directory?

**Comment by vovkkk** [comment]
> not automatically set the project folder

When you browse any folder, press <kbd>p</kbd>.

> Or don't automatically add the folder if it is larger than a certain size

Counting size would take some time, bigger size more time, approx. 1 second per 1000 files.

**Comment by excetara2** [comment]
```
	{
		"keys": ["super+k","super+l"],
		"command": "dired_jump",
		"args": { "new_window": "auto"}
		},
```

This is what I mean for the command so I don't hit p. It brings up a quick panel and then opens a completely new window. When the new window opens it seems to automatically add the folder jumped to as the project root. I guess you just have to hit `f` to tell it not to be the root anymore and hopefully it will stop trying to load the files up and behave sluggishly. I will check when I get another large usb drive again.

**Comment by vovkkk** [comment]
Yeah, I know. My point was that when you press <kbd>p</kbd> in existing browser, then it jumps without adding folder to project.
